src/test/java/org/folio/rest/impl/EholdingsProvidersImplTest.java

```diff
@@ -462,15 +462,6 @@ private Provider getUpdatedProvider() {
               .withIncluded(false)))));
   }
 
-  private ExtractableResponse<Response> getResponseWithStatus(String resourcePath, int expectedStatus) {
```

Comment:
same question as above

Reply:
Moved the code to base class -- WireMockTestBase.java since it was used in multiple places